### ISSUE TITLE
Think of a better UX

### ISSUE THREAD

**Issue by obla** [ozfortress/citadel#197]
Teams + Rosters difference is a little confusing for some players

**Comment by abender777** [comment]
Agree, had a look, and thought of a few ideas.

Using https://warzone.ozfortress.com/teams/47 as example:

Change 1:
- Replace "no safeword playing in Premier for Season 18" with "Active roster for no safeword playing in Premier for season 18" or "Active roster playing in Premier for season 18"

Change 2:
-Replace "Team Roster" with "Inactive roster" - and only display players who are not in current rosters (i.e the list above).
-Could also make the inactive roster minimised (v arrow to expand).

OR

-Sort players in the "Team Roster" section to have the active players in a roster up the top.
-Add another identifier (like captain) next to players names saying "Active player"...or "Inactive Player" depending on preference.

**Comment by BenjaminSchaaf** [comment]
First change is great! @abender777 

Though I think either of the suggestions for the 2nd change would break for be confusing if/when multiple leagues are running in parallel. Maybe changing the title from "Team Roster" to "Player Pool" or something like that would be better?

**Comment by abender777** [comment]
Yeah good point, imagine if there was HL! "Player Pool" sounds like a good alternative.

So for first change, what is better? 
"Active roster for no safeword playing in Premier for season 18"
or
"Active roster playing in Premier for season 18"

**Comment by BenjaminSchaaf** [comment]
The first once, since teams can sign up with a different name to their team name

**Comment by obla** [comment]
I like it. Nice boys.

**Comment by BenjaminSchaaf** [comment]
How much of an issue is this still?

**Comment by obla** [comment]
With the merging of teams and rosters I think it is fine now. If more issues come up we can re-open / create a new issue.
